### PR TITLE
Prettify

### PR DESCRIPTION
**Pull request by alanssitis**

### CODE REVIEW

File: web/src/pages/team/wdobert.md [hunk 1]
```diff
@@ -164,7 +164,7 @@ _Figure 1: Test pattern applied to our LED prototyping hardware by the latest dr
...
+While developing the LED driver, I was simultaneously organzing the structure of our firmware application. Although I made important steps toward this goal last week, there was still some smoothing of rough edges to do. The entrypoint for our main application is now in _app_core.c_, and my intention is that any additional files that support our application in a central manner be named with \_app\__ prefix. A primary example of this is the debugging module I created (\_app_debug.c_). At the moment, this module contains a function to print timestamped log messages to the integrated serial interface, but it will be expanded in the future to contain firmware debugging functions as needed. Obtaining an accurate timestamp for log messages necessitated enabling a hardware timer and writing a basic interrupt handler for it. As with _driver_led.c_, my goal is for all driver source files to be prefixed with \_driver\_\_ as they are created for new hardware (such as small panel displays).
```
Comment (WDobert) [inline web/src/pages/team/wdobert.md:167]: A couple formatting errors:

image
__app_ vs. \_app\_\_
_app_debug.c_ vs. \_app\_debug.c\_

image
__driver_ vs. \_driver\_\_

**Review by WDobert — CHANGES_REQUESTED** [review]

File: web/src/pages/team/wdobert.md (hunk 1 above)
Comment (benjamin-owen) [reply web/src/pages/team/wdobert.md:167]: Bro change _this__ to `this` please

Comment (alanssitis) [reply web/src/pages/team/wdobert.md:167]: @WDobert follow @benjamin-owen's thing

**Review by WDobert — APPROVED** [review]

**Review by benjamin-owen — APPROVED** [review]
lgtm

**Pull request merged**